lib/inspec/resources/ipnat.rb

```diff
@@ -0,0 +1,58 @@
+require "inspec/resources/command"
+module Inspec::Resources
+  class IpNat < Inspec.resource(1)
+    name "ipnat"
+    supports platform: "freebsd"
```

Comment:
Should this be 'bsd', not freebsd' ? That would include freebsd

Reply:
Yes, that's right.
Output of the target instance. 
```
inspec> inspec.os.family
=> "bsd"
```
Will make the change.

Reply:
Made the required change!